gui.py

```diff
@@ -55,7 +55,8 @@
 
 
 def post(qs, data):
-    r = requests.post(settings["url"] + qs, data=data, cert=settings["cert"], verify=False)
+    headers = {'Content-Type': 'application/xml'}
```

Comment:
I can't verify this on my own right now. However, WeChat's official [debug tool](https://mp.weixin.qq.com/debug/cgi-bin/apiinfo?t=index&type=%E6%B6%88%E6%81%AF%E6%8E%A5%E5%8F%A3%E8%B0%83%E8%AF%95&form=%E6%96%87%E6%9C%AC%E6%B6%88%E6%81%AF) is sending requests with `text/xml`.

Are you sure the WeChat server sends requests with `application/xml`? This program should mimic WeChat server's behavior.